src/qpm.io/common/package.go

```diff
@@ -146,6 +170,21 @@ func LoadPackages(vendorDir string) (map[string]*PackageWrapper, error) {
 	return packageMap, err
 }
 
+func IsCopyLeftLicense(license msg.LicenseType) bool {
+    licenses := []msg.LicenseType{msg.LicenseType_GPL_2_0,
+                                  msg.LicenseType_GPL_3_0,
+                                  msg.LicenseType_LGPL_2_1,
+                                  msg.LicenseType_LGPL_3_0,
+    }
+    
+    for _,v := range licenses {
+        if (v == license) {
+            return true;
+        }
+    }
+    return false;
+}
+
 func (pw PackageWrapper) Save() error {
```

Comment:
We have decided that hardcoding this interpretation of licenses directly into the client wasn't really a good idea after all. We're going to implement something similar on the server by interpreting properties of the license that we get from the GitHub license API.